cristin-person/src/main/java/no/unit/nva/cristin/person/client/CristinPersonApiClient.java

```diff
@@ -117,20 +117,19 @@ public List<CristinPerson> getEnrichedPersonsWithNinUsingQueryResponse(HttpRespo
                 : combineResultsWithQueryInCaseEnrichmentFails(personsFromQuery, enrichedCristinPersons);
     }
 
+    @SuppressWarnings({"PMD.DataflowAnomalyAnalysis"})
```

Comment:
What is this pmd error in layman's terms?

Reply:
`bang your head against the wall`

if there are paths through your code that don't use a declarations, it complains.